plutus-extra/src/Plutus/V1/Ledger/Contexts/Extra.hs

```diff
@@ -194,6 +197,54 @@ allDatumsAtInput ::
   [datum]
 allDatumsAtInput = allDatumsAt False
 
+{-# INLINEABLE parseDatum #-}
+
+{- | Helper function to parse a UTXO's datum and keep the UTXO
```

Comment:
This requires an `@since`, as per the [style guide](https://github.com/mlabs-haskell/styleguide#documentation).